util/slice_utils.go

```diff
@@ -0,0 +1,17 @@
+package util
+
+import "sort"
+
+// IsStringPresentInSlice returns true if the specified target is present in the given slice. Requires the slice to
+// be ordered in ascending order, due to the underlying requirement with SearchStrings.
+func IsStringPresentInSlice(target string, slice []string) bool {
+	index := sort.SearchStrings(slice, target)
+
+	return index < len(slice) && slice[index] == target
```

Comment:
Yeah, I was really unsure about using `sort` too because of that "needs the slice to be sorted" requirement... All right, changed the implementation to a simple for loop.